sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-use azure_core::http::{ClientMethodOptions, ClientOptions, Etag};
-
-use crate::models::ThroughputProperties;
+use crate::{
+    constants::{self},
```

Comment:
This can actually be rewritten as just `constants`. You only need `constants::{self, ...}` when you're importing _both_ the `constants` module **and** things from within it. This kind of error can easily occur when you start out importing something and then switch to just referencing it using the `constants::` module prefix :).

```suggestion
    constants,
```